ros/angel_system_nodes/angel_system_nodes/intent_detector.py

```diff
@@ -0,0 +1,140 @@
+import json
+import time
+
+from cv_bridge import CvBridge
+import cv2
+import numpy as np
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import Header
+
+from angel_msgs.msg import InterpretedAudioUserIntent, Utterance
+
+# Please refer to labels defined in
+# https://docs.google.com/document/d/1uuvSL5de3LVM9c0tKpRKYazDxckffRHf7IAcabSw9UA .
+NEXT_STEP_KEYPHRASES = ['skip', 'next step']
+PREV_STEP_KEYPHRASES = ['prev step', 'last step', 'go back']
```

Comment:
Maybe could have these keyphrases as a config file passed into this node. 

Locally, I added `next`, `previous`, and `previous step`.

Reply:
Do you have an example on the convention to follow for node configs? 

The one concern for investing effort into that is that entire node implementation is only an interim hacky fix. The end goal is a classifier that would entail us removing these keyphrases altogether. If it'll make things go more smoothly in the demo, I can try setting up a config.

Reply:
You could take a look at the config for the intent_to_cmd node, which uses this simple yaml config https://github.com/PTG-Kitware/angel_system/blob/master/config/angel_system_cmds/user_intent_to_sys_cmd_v1.yaml. Not super high priority though.

I would definitely add "previous" and "next" to the key phrases list though, if you can't get to the config.